### PR TITLE
Move the PRF-output length check into copyPrfOutput

### PR DESCRIPTION
**Pull request by mmv08**
From the simplification review: the 32-byte PRF-output check was duplicated verbatim at both `copyPrfOutput` call sites (`createPasskey` and `getPasskeyPrfOutput`). `copyPrfOutput` already exists to normalize and validate authenticator PRF results and already owns the `PRF_UNAVAILABLE` boundary, so the length check moves inside it and each call site shrinks to one line.

Observable behavior is unchanged: both paths threw `PRF_UNAVAILABLE` / "PRF output must be 32 bytes" before and still do. The `@throws` docs on the helper and both callers now state the malformed-output case.

Tests updated alongside: `copyPrfOutput` unit tests use 32-byte fixtures, the boundary-values case (0/255) folds into the plain-array test, and a new test pins the not-32-bytes rejection for views, buffers, and arrays.

Full suite passes (51/51).

**Stack 6/16** — based on #28; merge after it.

### CODE REVIEW

File: test/passkey.test.ts [hunk 1]
```diff
@@ -8,51 +8,64 @@ const ORIGINAL_NAVIGATOR = Object.getOwnPropertyDescriptor(
...
+  expectError(
+    () => copyPrfOutput(withFirstValue(Number.NaN)),
+    "PRF_UNAVAILABLE",
+  );
+});
+
+test("copyPrfOutput rejects PRF output that is not 32 bytes", () => {
+  expectError(() => copyPrfOutput(new Uint8Array(31)), "PRF_UNAVAILABLE");
+  expectError(() => copyPrfOutput(new ArrayBuffer(33)), "PRF_UNAVAILABLE");
 });
```
[the remark this answers is not in the document]
Comment (mmv08) [reply test/passkey.test.ts:71]: Good catch — added a plain `number[]` case ([1, 2, 3], valid byte values so it fails on length rather than element validation). The shared length check sits after all three normalization branches, so this pins the array-like branch against a refactor back to per-branch early returns.
